signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java

```diff
@@ -606,7 +606,7 @@ private void setIsEmpty(boolean newValue) {
     private void ensureSignatureBitmap() {
         if (mSignatureBitmap == null) {
             mSignatureBitmap = Bitmap.createBitmap(getWidth(), getHeight(),
-                    Bitmap.Config.ARGB_8888);
+                    Bitmap.Config.RGB_565 );
```

Comment:
Why this? In that case the generated bitmap will not have the alpha channel and we are going to sacrifice the transparency.